charts/celestia-app/values.yaml

```diff
@@ -289,24 +289,24 @@ app:
         address: ":8080"
         blockchain: app
         network: network
-        retries: "3"
+        retries: 3
```

Comment:
Did we also already adjust the script that creates this yaml?

Reply:
Also, sometimes the consensus application handles variables strage. Even if it's a number, sometimes the tolm expects a string. Just make sure everything works as expected and the application actually reads the values from the configuration.